types/video-react/index.d.ts

```diff
@@ -0,0 +1,510 @@
+         * use the <source> element within the Player block to specify the
+         * video to embed.
+         */
+        src?: string;
+        /**
+         * A URL indicating a poster frame to show until the user plays or
+         * seeks. If this attribute isn't specified, nothing is displayed
+         * until the first frame is available; then the first frame is shown
+         * as the poster frame.
+         */
+        poster?: string;
+        /**
+         * This enumerated attribute is intended to provide a hint to the
+         * browser about what the author thinks will lead to the best user
+         * experience. It may have one of the following values:
+         *
+         * - none: indicates that the video should not be preloaded.
+         * - metadata: indicates that only video metadata should be preloaded.
+         * - auto: indicates that both video and audio should be preloaded.
+         * (even if the user is not interacting with the video)
+         */
+        preload?: 'none' | 'metadata' | 'auto';
+        /**
+         * A Boolean attribute which indicates the default setting of the audio
+         * contained in the video. If set, the audio will be initially silenced.
+         * Its default value is false, meaning that the audio will be played when
+         * the video is played.
+         */
+        muted?: boolean;
+        /**
+         * [iOS only] Determines whether HTML5 videos play inline or use the native
+         * full-screen controller.
+         */
+        playsInline?: boolean;
+        /**
+         * The aspect ratio is the width of the video divided by its height.
+         * Possible values:
+         *
+         * - auto
+         * - 16:9
+         * - 4:3
+         */
+        aspectRatio?: string;
+        /**
+         * If specified, the video automatically begins to play back as soon as
+         * it can do so without stopping to finish loading the data.
+         */
+        autoPlay?: boolean;
+
+        /**
+         * Seek the Video at A Specific Time On Load
+         */
+        startTime?: number;
+    }
+
+    interface ShortcutItem {
+        /**
+         * The key code for the shortcut.
+         */
+        keyCode: number;
+        /**
+         * Defines if the action fires when the CTRL
+         * key is pressed.
+         */
+        ctrl: boolean;
+        /**
+         * The function to control the player when
+         * the shortcut is pressed.
+         */
+        handle: (state: PlayerState, actions: PlayerActions) => void;
+    }
+
+    interface ShortcutProps {
+        /**
+         * Allow click to play/pause, default: `true`
+         */
+        clickable?: boolean;
+        /**
+         * Allow double click to toggle fullscreen state, default: `true`
+         */
+        dblclickable?: boolean;
+        /**
+         * Add your own shortcuts
+         */
+        shortcuts?: Array<ShortcutItem>;
```

Comment:
The syntax `type[]` should be used instead of `Array<type>` for simple types

```suggestion
        shortcuts?: ShortcutItem[];
```